docs/ingestion/compaction.md

```diff
@@ -118,14 +118,14 @@ To perform a manual compaction, you submit a compaction task. Compaction tasks m
 
 |Field|Description|Required|
 |-----|-----------|--------|
-|`type`|Task type. Should be `compact`|Yes|
-|`id`|Task id|No|
-|`dataSource`|Data source name to compact|Yes|
+|`type`|Task type. Set the value to `compact`.|Yes|
+|`id`|Task ID.|No|
+|`dataSource`|Data source name to compact.|Yes|
```

Comment:
Nit: probably no need for a period in a table like this. I've gone back and forth. In the end, I don't think they add anything.

Maybe change "Set the value to" to the shorter "Must be", or just show the literal value.

Reply:
Removed end punctuation in descriptions that consist of one or two words and don't contain a call to action.